marshaling_test.go

```diff
@@ -256,27 +256,210 @@ func (s *Zuite) TestStructScan_skipFieldsWithNoTag() {
 	require.Equal(s.T(), "ignore me", data.Ignore)
 }
 
-func (s *Zuite) TestStructScan_slicesNotSupported() {
-	ws := s.defs.MustNewWorksheet("all_types")
+func (s *Zuite) TestStructScan_slices() {
```

Comment:
sweet test! now the `nil`'ness of slices we discussed this am makes that much more sense :)